manifest.json

```diff
@@ -1,29 +1,35 @@
 {
   "name": "SQL Importer",
-  "description": "Import SQL data from your own database",
+  "description": "Import data from your SQL databases",
   "picture": "picture.png",
-  "version": "0.2.3",
+  "version": "0.3.0",
   "tags": [
     "incoming",
     "oneColumn"
   ],
   "private_settings": [
     {
+      "type": "string",
```

Comment:
@sbellity and @michaloo, can we force this somehow after we have published it to production?